lib/install/tailwind/application.tailwind.css

```diff
@@ -1,3 +1 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
```

Comment:
It could be useful to open a separate PR just for this change, in case #164 takes a while to get merged. 

This would be useful right away, regardless of how you install tailwind. Those installation woes can be worked around in various ways, but this one can't be - you just have to replace it every time, all the time.

The only question is: do we want to be backward compatible with 3.x? I don't think so (since all the generators I'm aware of are using `tailwindcss@latest`, which is 4.x now)

However, if backward compatibility is not a concern, I don't see why this couldn't be accepted right away as a separate PR.